codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java

```diff
@@ -82,6 +90,9 @@
                     }
                     baos.write(node.symbol);
                     node = root;
+                    trailingPadBits = bits;
```

Comment:
it appears not in the current code ... it seems like the terminal nodes are spread amongst multiple leaf nodes ... so no null children nodes but it seems like it is currently decoding invalid codes because of this ... @jpinner ... any thoughts?